packages/core-ethereum/src/hashedSecret.spec.ts

```diff
@@ -346,57 +325,4 @@ describe('test hashedSecret', function () {
       }
     })
   })
-
-  describe('integration', function () {
```

Comment:
Not sure whether we should drop integration tests

Reply:
I plan to rewrite these tests as they don't adequately test the code, and the requirements make them very slow.